test/test_mps.py

```diff
@@ -10276,11 +10270,76 @@ def get_samples():
                 self.assertEqual(cpu_out, mps_out, atol=atol, rtol=rtol)
 
             except Exception as e:
-                if any(s in str(e).lower() for s in ["int64", "float16", "div truc rounding"]):
-                    self.skipTest(f"Expected Runtime Error: {str(e)}")
+                raise e
+
+            if not (dtype.is_floating_point or dtype.is_complex):
+                # Maybe we should error here instead?
+                continue
```

Comment:
What is this continue doing? Skipping all samples but the first one?

Reply:
Removed that.